### PR TITLE
Feature/multiple managers

### PR DESCRIPTION
**Pull request by edward3sc**
Updating endpoints that use teams to get managers.

### CODE REVIEW

File: app/src/routes/v3/layer.router.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+const V3TeamService = require("../../services/v3TeamService");
+const V3LayerService = require("../../services/v3LayerService");
+
+const router = new Router({
+  prefix: "/contextual-layer"
+});
+
+class Layer {
+  static get middleware() {
+    return [UserMiddleware.mapAuthToUser];
```
Comment (owenjs) [inline app/src/routes/v3/layer.router.js:20]: This is neat

File: app/src/routes/v3/layer.router.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      logger.error(e);
+      ctx.throw(500, "Team retrieval failed.");
+    }
+    // get list of user teams
+    let teams = await V3TeamService.getUserTeams(ctx.request.body.user.id);
+    let matchingTeam = teams.find(obj => obj.id === owner.id);
+    const isManager =
+      matchingTeam &&
+      matchingTeam.attributes &&
+      (matchingTeam.attributes.userRole.toString() === "manager" ||
```
Comment (owenjs) [inline app/src/routes/v3/layer.router.js:39]: could write `matchingTeam?.attributes?.userRole?.toString() === "manager" || matchingTeam?.attributes?.userRole?.toString() === "administrator"`

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Comment (edward3sc) [reply app/src/routes/v3/layer.router.js:39]: I've never been able to get this to work. Always throws an error

Details:

    /opt/fw-contextual-layer/app/src/routes/v3/layer.router.js:37
          (matchingTeam?.attributes?.userRole?.toString() === "manager" ||
                        ^

    SyntaxError: Unexpected token .

Comment (owenjs) [reply app/src/routes/v3/layer.router.js:39]: That's annoying, no worries then!

**Review by owenjs — APPROVED** [review]

**Pull request merged**